tests/queries/0_stateless/01744_fuse_sum_count_aggregate.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS fuse_tbl;
+CREATE TABLE fuse_tbl(a Int8, b Int8) Engine = Log;
+INSERT INTO fuse_tbl SELECT number, number + 1 FROM numbers(1, 20);
+
+set optimize_fuse_sum_count_avg = 1;
+SELECT sum(a), sum(b), count(b) from fuse_tbl;
```

Comment:
Let's add at least a test where the fusion doesn't happen at all, e.g. `select sum(a), count(b)`.

Reply:
ok.